v4/core/base_service_test.go

```diff
@@ -30,12 +32,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-type Foo struct {
```

Comment:
Due to the addition of the build flags, I moved common test-related definitions into a new file to be shared among the tests.